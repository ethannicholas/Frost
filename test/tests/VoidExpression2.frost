method test() {
    Console.printLine().doSomethingElse()
}
