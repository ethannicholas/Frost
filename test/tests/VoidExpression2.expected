VoidExpression2.frost:2:5: error: method frost.io.Console.printLine() does not return a value
1 error
